content/en/docs/instrumentation/python/getting-started.md

```diff
@@ -265,21 +281,122 @@ service:
             processors: [batch]
 ```
 
-Then start the Docker container:
+Then run the docker command to acquire and run the collector based on this configuration:
 
-```sh
+```
 docker run -p 4317:4317 \
     -v /tmp/otel-collector-config.yaml:/etc/otel-collector-config.yaml \
     otel/opentelemetry-collector:latest \
     --config=/etc/otel-collector-config.yaml
 ```
 
-Install the OpenTelemetry Collector exporter:
+You will now have an OpenTelemetry Collector instance running locally.
+
+### Modify the code to export spans via OTLP
+
+The next step is to modify the code to send spans to the Collector via OTLP instead of the console.
 
-```sh
+To do this, install the OTLP exporter package:
+
+```
 pip install opentelemetry-exporter-otlp
 ```
 
-Finally, execute the following script:
+Then configure the exporter. By default, it will send to `locahost:4317`, which is what the collector listens on.
```

Comment:
I think worth mentioning how to set the OTLP endpoint via environment variables here as well? Or linking to a place that explains it

Reply:
I think I'll leave this as-is for now, we don't really have docs for this yet, and I think two things would need to happen as well:

1. Get this in the Exporters docs for each langauge
2. Decide on which convention we document as the primary way to do things (configure in code or as env vars), then adjust all docs to handle that preference and then also document the equally-valid alternative